samples/csharp/getting-started/Regression_AutoML/TaxiFarePrediction/Program.cs

```diff
@@ -0,0 +1,329 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using Common;
+using Microsoft.ML;
+using Microsoft.ML.AutoML;
+using Microsoft.ML.Data;
+using PLplot;
+using TaxiFarePrediction.DataStructures;
+
+namespace TaxiFarePrediction
+{
+    internal static class Program
+    {
+        private static string AppPath => Path.GetDirectoryName(Environment.GetCommandLineArgs()[0]);
+
+        private static string BaseDatasetsRelativePath = @"Data";
+        private static string TrainDataRelativePath = $"{BaseDatasetsRelativePath}/taxi-fare-train.csv";
+        private static string TestDataRelativePath = $"{BaseDatasetsRelativePath}/taxi-fare-test.csv";
```

Comment:
fyi -- the taxi-fare-train.csv / taxi-fare-test.csv datasets already exists in the repo and this is ok. 

Git won't store a second copy in the repo itself so the `git clone` will download quickly, though the duplicate files will take space on the users' disk as more than one copy will be pulled out of the git repo.

We could create a single dataset folder at the root of the repo, and point all samples at the unified location.

https://stackoverflow.com/questions/29947825/how-does-git-store-duplicate-files

Reply:
Huh, cool to know how git treats duplicate files

>We could create a single dataset folder at the root of the repo, and point all samples at the unified location.
This is what we've done. Used the datasets from here: https://github.com/dotnet/machinelearning-samples/tree/features/buildrel/datasets
Although... it looks like this folder was removed 8 days ago in master.... https://github.com/dotnet/machinelearning-samples/commit/3993b413594ad02e48e95b515c9bcb3962fa5bf7
Requires some further investigation....

Reply:
Have this sorted out... going to re-add datasets back to this folder, for current use by AutoML and potential future use by rest of samples repo. (ie, I think there may be some dataset duplication between C# and F# samples, etc)